facebook_scraper/utils.py

```diff
@@ -43,3 +46,36 @@ def make_html_element(html: str, url=DEFAULT_URL) -> Element:
 
 def html2text(html: str) -> str:
     return _html2text(html)
+
+
+month = r"Jan(?:uary)?|" \
+        r"Feb(?:ruary)?|" \
+        r"Mar(?:ch)?|" \
+        r"Apr(?:il)?|" \
+        r"May|" \
+        r"Jun(?:e)?|" \
+        r"Jul(?:y)?|" \
+        r"Aug(?:ust)?|" \
+        r"Sep(?:tember)?|" \
+        r"Oct(?:ober)?|" \
+        r"Nov(?:ember)?|" \
+        r"Dec(?:ember)?|" \
+        r"Yesterday|" \
+        r"Today"
+date = f"({month}) " + r"\d{1,2}"
```

Comment:
Why are you capturing the month?
Some dates include the year, like: `April 3, 2018 at 8:02 PM`
```suggestion
date = f"(?:{month}) " + r"\d{1,2}" + r"(?:, \d{4})?"
```